### PR TITLE
Update Tomas_Kanholm_BCO_000514.json

### PR DESCRIPTION
**Pull request by tkanholm**
My BCO is ready for final review before pFDA submission

### CODE REVIEW

File: Student_BCOs/Tomas_Kanholm_BCO_000514.json [hunk 1]
```diff
@@ -17,12 +33,95 @@
                     "createdBy"
                 ],
                 "email": "tkanholm@gwu.edu"
+            },
+            {
+                "name": "Giorgio Giurato",
+                "orcid": "https://orcid.org/0000-0002-0538-8978",
+                "affiliation": "Laboratory of Molecular Medicine and Genomics, Department of Medicine and Surgery, University of Salerno, via Allende, 1, Salerno, Baronissi, Italy",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Tomas_Kanholm_BCO_000514.json]: Please add "derivedFrom" to the contribution type --> see #112

File: Student_BCOs/Tomas_Kanholm_BCO_000514.json [hunk 2]
```diff
@@ -17,12 +33,95 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": "Maria Rosaria De Filippo",
+                "orcid": "https://orcid.org/0000-0001-8350-2599",
+                "affiliation": "Division of Molecular Pathology and Medical Genomics, “SS. Giovanni di Dio e Ruggi d’Aragona – Schola Medica Salernitana” University of Salerno",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Tomas_Kanholm_BCO_000514.json]: Please add "derivedFrom" to the contribution type --> see #112

File: Student_BCOs/Tomas_Kanholm_BCO_000514.json [hunk 3]
```diff
@@ -17,12 +33,95 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": "Antonio Rinaldi",
+                "orcid": "https://orcid.org/0000-0002-3087-2649",
+                "affiliation": "Laboratory of Molecular Medicine and Genomics, Department of Medicine and Surgery, University of Salerno, via Allende, 1, Salerno, Baronissi, Italy",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Tomas_Kanholm_BCO_000514.json]: Please add "derivedFrom" to the contribution type --> see #112

File: Student_BCOs/Tomas_Kanholm_BCO_000514.json [hunk 4]
```diff
@@ -17,12 +33,95 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": "Adnan Hashim",
+                "orcid": "https://orcid.org/0000-0003-4736-8315",
+                "affiliation": "Laboratory of Molecular Medicine and Genomics, Department of Medicine and Surgery, University of Salerno, via Allende, 1, Salerno, Baronissi, Italy",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Tomas_Kanholm_BCO_000514.json]: Please add "derivedFrom" to the contribution type --> see #112

File: Student_BCOs/Tomas_Kanholm_BCO_000514.json [hunk 5]
```diff
@@ -17,12 +33,95 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": "Giovanni Nassa",
+                "orcid": "https://orcid.org/0000-0001-7453-1240",
+                "affiliation": "Laboratory of Molecular Medicine and Genomics, Department of Medicine and Surgery, University of Salerno, via Allende, 1, Salerno, Baronissi, Italy",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Tomas_Kanholm_BCO_000514.json]: Please add "derivedFrom" to the contribution type --> see #112

File: Student_BCOs/Tomas_Kanholm_BCO_000514.json [hunk 6]
```diff
@@ -17,12 +33,95 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": "Maria Ravo",
+                "orcid": "https://orcid.org/0000-0003-1003-2922",
+                "affiliation": "Laboratory of Molecular Medicine and Genomics, Department of Medicine and Surgery, University of Salerno, via Allende, 1, Salerno, Baronissi, Italy",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Tomas_Kanholm_BCO_000514.json]: Please add "derivedFrom" to the contribution type --> see #112

File: Student_BCOs/Tomas_Kanholm_BCO_000514.json [hunk 7]
```diff
@@ -17,12 +33,95 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": "Francesca Rizzo",
+                "orcid": "https://orcid.org/0000-0001-9705-2461",
+                "affiliation": "Laboratory of Molecular Medicine and Genomics, Department of Medicine and Surgery, University of Salerno, via Allende, 1, Salerno, Baronissi, Italy",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Tomas_Kanholm_BCO_000514.json]: Please add "derivedFrom" to the contribution type --> see #112

File: Student_BCOs/Tomas_Kanholm_BCO_000514.json [hunk 8]
```diff
@@ -17,12 +33,95 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": "Roberta Tarallo",
+                "orcid": "https://orcid.org/0000-0001-9668-3632",
+                "affiliation": "Laboratory of Molecular Medicine and Genomics, Department of Medicine and Surgery, University of Salerno, via Allende, 1, Salerno, Baronissi, Italy",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Tomas_Kanholm_BCO_000514.json]: Please add "derivedFrom" to the contribution type --> see #112

File: Student_BCOs/Tomas_Kanholm_BCO_000514.json [hunk 9]
```diff
@@ -17,12 +33,95 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": "Alessandro Weisz",
+                "orcid": "https://orcid.org/0000-0003-0455-2083",
+                "affiliation": "Laboratory of Molecular Medicine and Genomics, Department of Medicine and Surgery, University of Salerno, via Allende, 1, Salerno, Baronissi, Italy",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Tomas_Kanholm_BCO_000514.json]: Please add "derivedFrom" to the contribution type --> see #112

File: Student_BCOs/Tomas_Kanholm_BCO_000514.json [hunk 10]
```diff
@@ -17,12 +33,95 @@
...
+        "Pipeline for high-throughput analysis of small non-coding RNA data obtained by smallRNA-seq",
+        "Adapted from Giurato, Giorgio, Maria Rosaria De Filippo, Antonio Rinaldi, Adnan Hashim, Giovanni Nassa, Maria Ravo, Francesca Rizzo, Roberta Tarallo, and Alessandro Weisz. “IMir: An Integrated Pipeline for High-Throughput Analysis of Small Non-Coding RNA Data Obtained by SmallRNA-Seq.” BMC Bioinformatics 14 (December 13, 2013): 362. https://doi.org/10.1186/1471-2105-14-362.",
+        "This is single integrated pipeline software (iMIR v 1.0.0) for for high-throughput analysis of small non-coding RNA data obtained by smallRNA-seq. Description domain pipeline steps are named after the buttons a user would press to execute those steps.",
+        "Note: The software version for some execution domain software prerequisites were not listed in the iMir documentation. These include CutAdapt, SQUID, PDF::AP12, and Randfold.In cases where software version information was not available, the version number is listed as the latest version available as of the publication date of the paper (December 13, 2013)"
     ],
     "extension_domain": {},
     "description_domain": {
```
Comment (jpat1546) [inline Student_BCOs/Tomas_Kanholm_BCO_000514.json:136]: Are these step names correct? See #117

**Review by jpat1546 — CHANGES_REQUESTED** [review]
@tkanholm 
I've reviewed your updated BCO and made comments on what further needs to be fixed- I think just the description domain and provenance domain. Please make all the changes in the BCO editor, download, and update this pull request with the new JSON (copy/paste- same process as before). One all changes have been made, assign this pull request back to me and I will review again. Until then, assigning this pull request back to you while you make necessary changes.

File: Student_BCOs/Tomas_Kanholm_BCO_000514.json (hunk 10 above)
Comment (tkanholm) [reply Student_BCOs/Tomas_Kanholm_BCO_000514.json:136]: As per my [comment on issue #177](https://github.com/biocompute-objects/GW-SMHS-BIOC6223/issues/117#issuecomment-542297393), these step names are derived from the buttons a user would click to execute the required step in the integrated pipeline software my BCO is based on. Professor Keeney Ok'd this approach, so I think it should be ok? I also made a note to that effect in the usability domain to help make it clear.

If you feel strongly about this please let me know how you think I should go about it!

Thanks!

Comment (jpat1546) [reply Student_BCOs/Tomas_Kanholm_BCO_000514.json:136]: Ahh yes. Thanks for reminding me @tkanholm ! No, this is fine.

**Review by jpat1546 — APPROVED** [review]
### Your BCO is approved
Great job fixing all your assigned issues, @tkanholm  .

Prior to submitting to the pFDA challenge, please make the following 2 changes to your BCO in the BCO Editor (then download and submit to the pFDA):
In Provenance domain:

- Change reviewer status from "in review" to "approved"
- Change reviewer_comment from "Reviewed by BioCompute technical assistant, approval pending modification by author" to "Reviewed and approved by BioCompute technical assistant"

Leave me a comment if these instructions are not clear. Thanks!

**Comment by tkanholm** [comment]
Awesome, thank you! Changed the review status and submitted to precisionFDA :)

**Pull request merged**